src/applications/mhv/secure-messaging/components/MessageThread/MessageThreadItem.jsx

```diff
@@ -27,7 +28,7 @@ const MessageThreadItem = props => {
     triageGroupName,
   } = message;
 
-  const isRead = readReceipt === 'READ';
+  const isSentOrRead = folderId === -1 || readReceipt === 'READ';

```

Comment:
I'm curious about what this -1 means? If the folder is is -1, the message is read? I think it might make sense to have a constant for this value, especially if it's used anywhere else. 🙂

Reply:
I'm also curious if there are any unit tests to confirm the correct read/unread behavior.

Reply:
> I think it might make sense to have a constant for this value

Great point. @KyleCardwell we are keeping system folders properties in `constants.js` where you can utilize this value from

Reply:
Great idea.  I'll add a new constant and update to use the existing folderId constant